zerver/lib/test_runner.py

```diff
@@ -130,7 +130,7 @@ class Process(multiprocessing.context.ForkProcess):
         daemon = cast(bool, property(lambda self: False, lambda self, value: None))
 
 
-django_runner.multiprocessing = NoDaemonContext()
+setattr(django_runner, "multiprocessing", NoDaemonContext())
```

Comment:
Should we be subclassing `django_runner` instead of doing this hack?

Reply:
`django_runner` is a module, we can't subclass it.

Reply:
We can get rid of this! #22623